frontend/src/bundles/auth/auth-api.ts

```diff
@@ -33,7 +33,7 @@ class AuthApi extends HttpApiBase {
             },
         );
 
-        return await response.json<UserSignUpResponseDto>();
+        return response.json<UserSignUpResponseDto>();
```

Comment:
we don't need this changes

Reply:
A few blocks above Maksym told that it`s better to return values without await:
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/88#discussion_r1304760618

Reply:
@ptarho  It's all good Paul. Nikita says to keep await for returns in frontend while in backend no need.